### ISSUE TITLE
Create hover on heading of faq's page.

### ISSUE THREAD

**Issue by ads-22** [dsc-jiit-128/Hacktoberfest_Website#18]
heading's of FAQ's page 

1. Rules
2. General
2. Shipping

Improve UI of FAQ's page by creating hover over sub-headings.

**Comment by shravani05** [comment]
Can I work on this issue? @ads-22

**Comment by ads-22** [comment]
Yup @shravani05  go ahead.. 👍

**Comment by ampsteric** [comment]
I would like to work on this issue as well. please assign in case the issue is not resolved.

**Comment by ads-22** [comment]
@shravani05 you have time till 2st october to make PR.

**Comment by ads-22** [comment]
@ampsteric you have been added to queue😀

**Comment by shravani05** [comment]
Yes, I am working on the issue. I'll make the PR as soon as possible! @ads-22

**Comment by ampsteric** [comment]
sir I have made a PR. its my humble request to you to please accept it on 1st october as I am participating in hacktober fest and so it can also be counted .

**Comment by shravani05** [comment]
@ampsteric this issue has already been assigned to me. I am in the middle of solving this issue. And as to make it valid for Hacktoberfest, I am going to make the PR on 1 Oct.
Would you kindly look for some other issue that has not been assigned to anyone?

**Comment by shravani05** [comment]
@ads-22 Currently, I have added the hover effect which changes the color of the subheadings to yellow on hover. What other effects do you want me to add on hover? Or if you want any other specific color, please mention that as well. 
![hover](https://user-images.githubusercontent.com/43898702/94564374-89c18880-0285-11eb-8cc0-9f8fc01640cf.png)

**Comment by ampsteric** [comment]
@shravani05 I understand your point but as @ads-22 has also assigned this issue to me, I am also eligible to work.
@ads-22  I have already made a PR resolving the above mentioned issue please accept it after 1st october so that it can be counted as my work for hacktober fest also.

your faithfully,
ampsteric

**Comment by shravani05** [comment]
@ads-22 I have made the PR with necessary requirements, please review it.

**Comment by ampsteric** [comment]
@ads-22 I had already opened a PR 2 days back. Please review it.
#31

**Comment by ampsteric** [comment]
#31

**Comment by Castella1313** [comment]
@ampsteric you aren't assigned this issue, so closing your pr

**Comment by ampsteric** [comment]
https://github.com/dsc-jiit-128/Hacktoberfest_Website/issues/18#issuecomment-700509084

please refer to above link to see that @ads-22 had assigned me this issue and told me to work.
please review and accept my PR.

**Comment by Castella1313** [comment]
@ampsteric you were added to queue, not assigned !!

**Comment by Castella1313** [comment]
@ads-22 has also mentioned that @shravani05 has time till 2nd oct, if not made a pr by @shravani05 till 2nd oct, it will be assigned to you

**Comment by ampsteric** [comment]
ok then.
but please atleast dont let my hardwork go to waste.
please dont mark my PR as invalid for the hactoberfest.

**Comment by Castella1313** [comment]
@ampsteric its always important to read the readme first of any repository , we have mentioned each and everything there.
